src/e2ee/RustEngine.ts

```diff
@@ -123,9 +124,7 @@ export class RustEngine {
             const keysClaim = await this.machine.getMissingSessions(members);
             if (keysClaim) {
                 await this.processKeysClaimRequest(keysClaim);
-                if (this.keyBackupVersion !== undefined) {
-                    await this.machine.backupRoomKeys();
-                }
+                this.backupRoomKeysIfEnabled();
```

Comment:
The missing `await` is intentional, as the idea here is to fire off the backup task asynchronously.

This could admittedly use a comment to explain that, though.